templates/hostalive.ini

```diff
@@ -26,7 +26,7 @@ areaAlpha = "0.5"
 areaMode = "all"
 lineWidth = "2"
 min = "0"
-yUnitSystem = "none"
+yUnitSystem = "msec"
```

Comment:
Are you sure about this?

Reply:
Sure, hostalive results are usually in msecs?
See screenshot from my tests, looks good and correct for me?

Reply:
This is the packet loss which is in %.

Reply:
@Al2Klimov thanks, sorry, mixed both graphs. Fixed in 2714dbf23d22b7195e04a77b9fa3275e3cdfc62b.